website/docs/cloud-docs/workspaces/settings/vcs.mdx

```diff
@@ -4,82 +4,63 @@ page_title: VCS Connections - Workspaces - Terraform Cloud and Terraform Enterpr
+#### Always trigger runs
 
-  For example, if you specify `/submodule/**/*.tf`, Terraform cloud will trigger runs for all the files changed nested below the `submodule` directory which end up with the `.tf` extension.
-  You could specify even more complex patterns such as `/**/networking/**/*` which will match all the files changed which have `networking` folder in the path, e.g. `/submodule/service-1/networking/private/main.tf`.
-  To learn more about glob patterns check [TODO: create glob patterns page]
-- **Specify trigger prefixes.** Terraform Cloud will queue runs for changes in any of the specified trigger directories matching the provided prefixes (including the working directory).
+This option instructs Terraform to begin a run when changes are pushed to any file within the repository. This can be useful for repositories that do not have multiple configurations but require a working directory for some other reason. However, we do not recommend this approach for true monorepos, as it queues unnecessary runs and slows down your ability to provision infrastructure.
 
-  For example, if you use a top-level `modules` directory to share Terraform code across multiple configurations, changes to the shared modules are relevant to every workspace that uses that repo. You can add `modules` as a trigger directory for each workspace to make sure they notice any changes to shared code.
-- **Mark the entire repository as relevant.** If you set the "Automatic Run Triggering" setting to "Always Trigger Runs," Terraform Cloud will assume that anything in the repository might affect the workspace's configuration, and will queue runs for any change.
+#### Only trigger runs when files in specified paths change
 
-  This can be useful for repos that don't have multiple configurations but require a working directory for some other reason. It's usually not what you want for true monorepos, since it queues unnecessary runs and slows down your ability to provision infrastructure.
+This option instructs Terraform to begin new runs only for changes that affected specified files and directories. This behavior also applies to [speculative plans](/cloud-docs/run/#speculative-plans) on pull requests.
 
-### Error Handling for Run Triggers
+You can use trigger patterns and trigger prefixes in the **Add path** field to specify groups of files and directories.
+- **Trigger Patterns:** (Recommended) Use glob patterns to specify the files that should trigger a new run. For example, `/submodule/**/*.tf`, specifies all files with the `.tf` extension that are nested below the `submodule` directory. You can also use more complex patterns like `/**/networking/**/*`, which specifies all files in the `networking` folder at this level of the file path (e.g., `/submodule/service-1/networking/private/main.tf`). Refer to [TODO: create glob patterns page](link) for details.
```

Comment:
@matejrisek - Are glob patterns a well known construct? I'm wondering if we could link to external documentation about then rather than reinventing the wheel in our own documentation. But I defer to you. I don't know what the rules are/how much detail we'd need to give!

Reply:
It's been decided that we won't be linking external documentation for this part, there's [an Asana ticket](https://app.asana.com/0/1201659973165547/1202390263139034) to create documentation for glob patterns.

Glob patterns is a known construct but it's not standardized. We're only allowing for the subset of it.